README.md

```diff
@@ -1,113 +1,54 @@
+[![Patreon](https://img.shields.io/badge/dynamic/json?color=%23e85b46&label=Support%20us%20on%20Patreon&query=data.attributes.patron_count&suffix=%20patrons&url=https://www.patreon.com/api/campaigns/4762180)](https://www.patreon.com/bePatron?u=37261055)
 
-## Profiling
+----
+[OpenDiablo2](https://opendiablo2.com/) is an ARPG game engine in the same vein of the 2000's games, and supports playing Diablo 2.
 
-There are many profiler options to debug performance issues. These can be enabled by suppling the following command-line option and are saved in the `pprof` directory:
+The engine is written in Go and is cross platform.
 
-`go run . --profile=cpu`
+> The project does not ship with the assets or content required to play Diablo 2.
+You must have a legally purchased copy of [Diablo 2](https://us.shop.battle.net/en-us/product/diablo-ii) and its expansion [Lord of Destruction](https://us.shop.battle.net/en-us/product/diablo-ii-lord-of-destruction) installed on your computer in order to run that game on this engine.
 
-Available profilers:\
-`cpu` `mem` `block` `goroutine` `trace` `thread` `mutex`
+If you like to contribute to OpenDiablo2, please be so kind to read our [Contribution Policy](./docs/CONTRIBUTING.md) first.
 
-You can export the profiler output with the following command:\
-`go tool pprof --pdf ./OpenDiablo2 pprof/profiler.pprof > file.pdf`
+----
 
-Ingame you can create a heap dump by pressing `~` and typing `dumpheap`. A heap.pprof is written to the `pprof` directory.
+## Documentation
 
-You may need to install [Graphviz](http://www.graphviz.org/download/) in order to convert the profiler output.
+_Stay awhile and listen_ ...
 
-## Debugging
+### ⚡ Project Info
 
-### Layouts
+* 👉 **[Current Status](./docs/status.md)** 👈 - what you should focus on
+* [Roadmap](./docs/roadmap.md) - Planning ahead
+* Design - High-level overview of the OpenDiablo2 org and its projects
+* FAQ - Common questions from new people to the project
 
-Layouts can show their boundaries and other visual debugging information when they render. Set `layoutDebug` to `true` in `d2core/d2gui/layout.go` to enable this behavior.
+### ⭐ For Users
 
-![Example layout in debug mode](https://user-images.githubusercontent.com/1004323/85792085-31816480-b733-11ea-867e-291946bfff83.png)
+* Buy the official game from Blizzard
+* Locate the MPQ files
+* Install OpenDiablo2 to your system (Linux/Windows/MacOS)
+* [Run it](./docs/play.md) - How to play the game
 
-## Roadmap
+### 🔥 For Developers
 
-There is an in-progress [project roadmap](https://docs.google.com/document/d/156sWiuk-XBfomVxZ3MD-ijxnwM1X66KTHo2AcWIy8bE/edit?usp=sharing),
-which will be updated over time with new requirements.
+* [Building](./docs/building.md) - Instructions for building the project
+* [Development](./docs/development.md) - Instructions for developers who want to contribute
+* [Profiling](.docs/profiling.md) - Debug performance issues
+* [Debugging](.docs/debug.md) - Common errors and pitfalls
```

Comment:
why did you split docs dir into .docs and docs? IMO it should be one

Reply:
I am not following, where do you see `docs` ? Everything is into the `./docs` directory.

Reply:
<img width="674" alt="Screen Shot 2021-03-25 at 3 21 44 PM" src="https://user-images.githubusercontent.com/936440/112551235-e78ae500-8d7d-11eb-9567-805d59de9af8.png">

Reply:
ah crap. thanks for spotting this out! :D